src/utils/requests/UserRequests.js

```diff
@@ -1,23 +1,56 @@
+const { URL } = require("../../config")
+
 /**
  * @function logUser
  * @description Function used to wrap the login request while simply requiring credentials as params
  * @param {String} credentials.email
  * @param {String} credentials.password // no need to encrypt yet
  */
 export const login = async (credentials) => {
-    // should do something with credentials...
-    // hint: use GET
+
+    const response = await fetch(`${URL}/login`, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify(credentials),
+    });
+
+    return await checkStatus(response);
 };
 
 /**
- * @function logUser
- * @description Function used to wrap the register request while simply requiring credentials as params
+ * @function registerUser
+ * @description Function used to wrap the register request while simply requiring data for account registration
+ * @param {String} entries.firstName
+ * @param {String} entries.lastName
  * @param {String} entries.email
  * @param {String} entries.password // no need to encrypt yet
- * @param {String} entries.confirmPassword
- * @param {String} entries.etc... name, age, ...rest
  */
+
+/*
+{type: "cors", url: "https://billsnap-development.herokuapp.com/billsnap/register", redirected: false, status: 400, ok: false, …}
+*/
 export const register = async (entries) => {
-    // should do something with entries...
-    // hint: POST
-};
\ No newline at end of file
+
+    const response = await fetch(`${URL}/register`, {
+        method: "POST",
+        headers: {
+            "Content-Type": "application/json"
+        },
+        body: JSON.stringify(entries),
+    });
+
+    return await checkStatus(response);
+};
+
+async function checkStatus(response) {
+    const parsedResponse = await response.json();
```

Comment:
we could remove the async/await since this is only used on responses that have been resolved